Log.swift

```diff
@@ -9,62 +9,205 @@ import Foundation
 // logLevel allows you to specify the minimum log level to display messages for.
 // Example:  setting logLevel to .warn will allow .warn and .error log
 // messages to display and prevent .debug and .info messages from displaying.
-var logLevel: LogLevel = .debug
+// It can be updated in the debugger by typing `expr logLevel = Log.trace`
+var logLevel: LogLevel = Log.info
```

Comment:
Because I changed `LogLevel` to a protocol, and I'm setting the `Log` enum, I have to leave the `Log` in there. Alternatively, I could make `logLevel` a `Log` instead of a `LogLevel`. I used the protocol in case anyone ever wanted to implement and set their own. That seems unlikely, though.